wpilibc/src/main/native/include/frc/DutyCycleEncoder.h

```diff
@@ -187,9 +199,12 @@ class DutyCycleEncoder : public wpi::Sendable,
   void SetConnectedFrequencyThreshold(int frequency);
 
   /**
-   * Get the encoder value.
+   * Get the encoder value in rotations.
+   *
+   * @warning This will return inaccurate values for up to 2 seconds after this
+   * encoder is initialized unless SetAssumedFrequency() is used.
    *
-   * @return the encoder value scaled by the full range input
+   * @return the encoder value in rotations
```

Comment:
Why change this? If the full range input is 2pi, then the return value is radians, not rotations, for example

Reply:
whoops, I missed that you could pass a conversion factor while trying to make them consistent